docs/_partials/_using-multiple-promises.md

```diff
@@ -264,13 +258,27 @@ To recap the steps we took:
 
 This is only the beginning of working with Promises. Next you will learn how to write and update Promises, and in the final thoughts we will showcase the composability of Promises to further optimise this workflow from three requests down to one.
 
-## Tearing it all down {#teardown}
+## Cleanup environment {#cleanup}
 
-The next section in this tutorial requires a clean Kratix installation. Before heading to it, please clean up your environment by running:
+To clean up your environment you need to delete the Promises and the Resource Requests.
+Kratix will automatically cleanup any Resource Requests when the parent Promise is deleted.
```

Comment:
🙌  Nice! This is what I was suggesting on the last section, just to do a bit more of a nuclear clean up option. I think this may be a nicer experience for people.